tests/rbd_mirror/test_rbd_mirror_image_features.py

```diff
@@ -0,0 +1,216 @@
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_journal_image_feature(rbd_mirror, pool_type, **kw):
+    """Method to validate change of image feature in primary
+    got reflected onto secondary in Journal based RBD mirroring.
+    Args:
+        rbd_mirror: Object
+        pool_type: Replicated or EC pool
+        **kw: test data
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+
+    try:
+        log.info(
+            "Running RBD journal based mirroring test case for image features reflection"
+        )
+        config = kw.get("config")
+        mirror1, mirror2 = [
```

Comment:
I think you dont need to create mirror1 and mirror2 again, instead you can use directly
        mirror1 = rbd_mirror.get("mirror1")
        mirror2 = rbd_mirror.get("mirror2")

Reply:
thanks Done.